src/org/opendatakit/briefcase/ui/automation/TransferFormsTableView.java

```diff
@@ -58,8 +62,12 @@ public TransferFormsTableView(TransferFormsTableViewModel model, String[] header
     columns.getColumn(SELECTED_CHECKBOX_COL).setPreferredWidth(40);
     columns.getColumn(FORM_NAME_COL).setMinWidth(formNameDims.width + 25);
     columns.getColumn(FORM_NAME_COL).setPreferredWidth(formNameDims.width + 25);
-    columns.getColumn(STATUS_COL).setMinWidth(statusDims.width + 25);
-    columns.getColumn(STATUS_COL).setPreferredWidth(statusDims.width + 25);
+    columns.getColumn(PULL_COL).setMinWidth(pullDims.width + 25);
```

Comment:
I think that you could set the `minWidth` and `preferredWith` to a fixed value of `40`, like the checkbox in the first column.